pyproject.toml

```diff
@@ -56,6 +56,8 @@ dev = [
     "types-requests==2.32.0.20240622",
     "gradio==4.44.1",
     "boto3==1.34.145",
+    "pyroscope-io>=0.8.7",
```

Comment:
please lock version for both new packages

Reply:
Done